mobile-app/app/components/icons/assets/_Default.tsx

```diff
@@ -1,7 +1,78 @@
+import { RootState } from '@store'
+import { tokenSelectorByDisplaySymbol } from '@store/wallet'
 import randomColor from 'randomcolor'
-import Svg, { Circle, SvgProps, Text } from 'react-native-svg'
+import Svg, { Circle, SvgProps, Text, G, Path } from 'react-native-svg'
+import { useSelector } from 'react-redux'
 
 export function _Default (symbol: string): (props: SvgProps) => JSX.Element {
+  const tokenDetail = useSelector((state: RootState) => tokenSelectorByDisplaySymbol(state.wallet, symbol))
+
+  if (tokenDetail?.isLoanToken === true) {
```

Comment:
Can't this be just `tokenDetail?.isLoanToken`?

Reply:
the value of `tokenDetail` we get by filtering allTokens by `displaySymbol`. 
there is a chance of getting tokenDetail as undefined value (Unexpected any value in conditional) 
so for this, an explicit comparison or typecast is required.